storage/src/vespa/storage/persistence/filestorage/filestorhandlerimpl.cpp

```diff
@@ -945,6 +951,7 @@ FileStorHandlerImpl::Stripe::getNextMessage(uint32_t timeout, Disk & disk)
         PriorityIdx::iterator iter(idx.begin()), end(idx.end());
 
         while (iter != end && operationIsInhibited(guard, iter->_bucket, *iter->_command)) {
+            _metrics->inhibited[iter->_command->getLoadType()].inc();
```

Comment:
As discussed offline (well, technically online due to the current circumstances...!) the retry mechanics of `getNextMessage` means that this metric risks over-counting when attempting to dequeue a message.